drivers/veteran_confirmation/app/models/veteran_confirmation/va_check_history.rb

```diff
@@ -5,11 +5,34 @@
 ###
 
 module VeteranConfirmation
-  class Checker
+  class VaCheckHistory < GrdaWarehouseBase
+    self.table_name = :va_check_histories
+
+    # Response values
     CONFIRMED = 'confirmed'.freeze
     NOT_CONFIRMED = 'not confirmed'.freeze
 
-    def check(clients)
+    belongs_to :client, class_name: 'GrdaWarehouse::Hud::Client'
+
+    scope :recent, -> { order(check_date: :desc).limit(1) }
```

Comment:
This seems odd to have a limit in a scope.  I wonder if we could impose that on the other side and just rename the scope `most_recent_first` or similar.